packages/runtime/src/components/builtin/Slot/Slot.tsx

```diff
@@ -0,0 +1,21 @@
+'use client'
+
+import { ReactNode, useContext } from "react"
+import { PropsContext } from "../../../next/components/SlotProvider"
+
+type Props = {
+  children: ReactNode
+  showFallback: boolean
+}
+
+function Slot(props: Props) {
+  const ctx = useContext(PropsContext)
+
+  return (
+      <>
+        { props.showFallback ? ctx.fallback : props.children }
```

Comment:
Seems we still have spacing inconsistencies. You can run `pnpm format` in the monorepo to format all files.

We should add this to our CI workflow 🤔